test/k8sT/Updates.go

```diff
@@ -11,6 +11,20 @@ import (
 )
 
 var _ = Describe("K8sUpdates", func() {
+
+	// This test runs 8 steps as following:
+	// 1 - delete all pods. Clean cilium, this can be, and should be achieved by
+	// `clean-cilium-state: "true"` option that we have in configmap
+	// 2 - install cilium `cilium:v1.1.4`
```

Comment:
Nit: this comment will go out of date quickly once we update the existing latest released version. What about "the latest version specified in `test/helpers/cons.go:CiliumStableVersion`"?

Reply:
This should be an overview of what the test is doing, it's easier to understand the test logic that read the test line by line.